src/components/Logo/AssetLogo.tsx

```diff
@@ -0,0 +1,60 @@
+import { SupportedChainId } from 'constants/chains'
+import useTokenLogoSource from 'hooks/useAssetLogoSource'
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import { MissingImageLogo } from './BaseLogo'
+
+const LogoImage = styled.img<{ size: string }>`
+  width: ${({ size }) => size};
+  height: ${({ size }) => size};
+  background: radial-gradient(white 60%, #ffffff00 calc(70% + 1px));
+  border-radius: 50%;
+  -mox-box-shadow: 0 0 1px white;
+  -webkit-box-shadow: 0 0 1px white;
```

Comment:
I thought prefixed properties were added by styled. Have you tested this?

Reply:
I copied this over from the existing code. I'll test it out

Reply:
yeah seems like styled components does this for us, thx!!